frontend/packages/core/src/Feedback/toast.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Snackbar } from "@material-ui/core";
+
+import type { AlertProps } from "./alert";
+import { Alert } from "./alert";
+
+export interface ToastProps extends AlertProps {
+  duration?: number;
+  onClose?: () => void;
+}
+
+const Toast: React.FC<ToastProps> = ({
+  onClose,
+  severity = "warning",
+  duration = 6000,
+  ...props
+}) => {
+  const [open, setOpen] = React.useState(true);
+
+  const onDismiss = () => {
+    if (onClose !== undefined) {
+      onClose();
+    }
+    setOpen(false);
+  };
+
+  return (
+    <Snackbar
+      open={open}
+      autoHideDuration={duration}
+      onExiting={onDismiss}
+      onClose={(_, reason) => {
+        // This way it will not auto close when clicking in the window, will instead wait on the timeout or onClose
+        if (reason !== "clickaway") {
+          setOpen(false);
+        }
+      }}
+      anchorOrigin={{ vertical: "bottom", horizontal: "right" }}
```

Comment:
Should we make this configurable? Wondering if we always want this in the bottom right

Reply:
Yeah it would probably be a good idea to make it customizable, and @material-ui already types it as well, so its easy

Reply:
Extended the type from snackbarprops so its fully customizable but still defaulted